### DISCUSSION TITLE
Python coding

### DISCUSSION THREAD

**Discussion by Jing-Pan-China** [unpackAI/DL201#9] [Q&A]
Hi, I have a question about in indexing_concepts_book notebook. 
https://github.com/unpackAI/DL201/blob/main/Week-1/1_2_Indexing_Concepts_Book.ipynb.

In Section 1: Indexing Tabular Data, why we need to transform startupData.columns to pandas Series format, can we use startupData.columns.str.strip() directly to  remove the whitespace? Thanks.

![111](https://user-images.githubusercontent.com/91070643/156355910-705c067e-4a82-43ef-9da3-f8133ad69274.jpeg)

**Comment by lenismerino** [comment]
Yes, that is also possible,  your suggestion allows to perform the same operation without creating a Series object.
